Add doc for `show_indexes`, `show_official_extensions` (#339)
kuzudb/kuzu-docs: src/content/docs/cypher/query-clauses/call.md

| `SHOW_WARNINGS()` | returns the contents of the [Warnings Table](/import#warnings-table-inspecting-skipped-rows) |
| `CLEAR_WARNINGS()` | clears all warnings in the [Warnings Table](/import#warnings-table-inspecting-skipped-rows)  |
| `TABLE_INFO('tableName')` | returns metadata information of the given table                                              |

> | `SHOW_OFFICIAL_EXTENSIONS` | returns all official [extensions](../../extensions) which can be installed by `INSTALL <extension_name>` |

Reviewer: ```suggestion
| `SHOW_OFFICIAL_EXTENSIONS` | returns all official [extensions](/extensions) which can be installed by `INSTALL <extension_name>` |
```

```cypher
CALL clear_warnings();
```

### SHOW_OFFICIAL_EXTENSIONS

> If you would like to know all official [extensions](../../extensions) available in kuzu, you can run the `SHOW_OFFICIAL_EXTENSIONS` function.

Reviewer: ```suggestion
If you would like to know all official [extensions](../../extensions) available in Kùzu, you can run the `SHOW_OFFICIAL_EXTENSIONS` function.
```

```

Output:
```
┌──────────┬─────────────────────────────────────────────────────────────────────────┐
│ name     │ description                                                             │
│ STRING   │ STRING                                                                  │
├──────────┼─────────────────────────────────────────────────────────────────────────┤
│ SQLITE   │ Adds support for reading from SQLITE tables                             │
│ JSON     │ Adds support for JSON operations                                        │
│ ICEBERG  │ Adds support for reading from iceberg tables                            │
│ HTTPFS   │ Adds support for reading and writing files over a HTTP(S)/S3 filesystem │
│ DELTA    │ Adds support for reading from delta tables                              │
│ POSTGRES │ Adds support for reading from POSTGRES tables                           │
│ FTS      │ Adds support for full-text search indexes                               │
│ DUCKDB   │ Adds support for reading from duckdb tables                             │
└──────────┴─────────────────────────────────────────────────────────────────────────┘
```

### SHOW_LOADED_EXTENSIONS

> If you would like to know information about loaded extensions in kuzu, you can run the `SHOW_LOADED_EXTENSIONS` function.

Reviewer: ```suggestion
If you would like to know information about loaded extensions in Kùzu, you can run the `SHOW_LOADED_EXTENSIONS` function.
```

| Column | Description | Type |
| ------ | ----------- | ---- |
| extension name | name of the extension | STRING |

> | extension source | whether the extension is official or developed by third-party | STRING |

Reviewer: ```suggestion
| extension source | whether the extension is officially supported by Kùzu Inc., or developed by a third-party | STRING |
```

| extension path | the path to the extension | STRING |

```cypher
CALL SHOW_LOADED_EXTENSIONS() RETURN *;
```

```
┌────────────────┬──────────────────┬─────────────────────────────────────────────────────────────────────────────┐
│ extension name │ extension source │ extension path                                                              │
│ STRING         │ STRING           │ STRING                                                                      │
├────────────────┼──────────────────┼─────────────────────────────────────────────────────────────────────────────┤

> │ FTS            │ USER             │ /Users/z473chen/Desktop/code/kuzu/extension/fts/build/libfts.kuzu_extension │

Reviewer: ```suggestion
│ FTS            │ USER             │ /path/to/kuzu/extension/fts/build/libfts.kuzu_extension │
```

Reviewer: @acquamarin Above, we state that the `extension source` indicates whether an extension is officially supported by Kùzu Inc., or developed by a third-party. However, when I run this command, I get the `FTS` extension source  as `USER`. Is this because I'm using a local extension build? What would it show if I downloaded the official extension? We should update this to show that instead.

```
┌────────────────┬──────────────────┬───────────────────────────────────────────┐
│ extension name │ extension source │ extension path                            │
│ STRING         │ STRING           │ STRING                                    │
├────────────────┼──────────────────┼───────────────────────────────────────────┤
│ FTS            │ USER             │ extension/fts/build/libfts.kuzu_extension │
└────────────────┴──────────────────┴───────────────────────────────────────────┘
```

> │ FTS            │ USER             │ /path/to/kuzu/extension/fts/build/libfts.kuzu_extension │

Reviewer: ```suggestion
│ FTS            │ OFFICIAL         │ extension/fts/build/libfts.kuzu_extension │
```
